internal/core/state/statefile/statefile.go

```diff
@@ -5,33 +5,43 @@ import (
 	"errors"
 	"fmt"
 	"net/http"
+	"os"
 	"path/filepath"
 	"sort"
 	"strings"
 
 	"github.com/deref/exo/internal/core/state/api"
 	state "github.com/deref/exo/internal/core/state/api"
 	"github.com/deref/exo/internal/deps"
+	"github.com/deref/exo/internal/gensym"
 	"github.com/deref/exo/internal/util/atom"
 	"github.com/deref/exo/internal/util/errutil"
 	"github.com/deref/exo/internal/util/pathutil"
 )
 
-func New(filename string) *Store {
+type Config struct {
+	StoreFilename    string
+	DeviceIDFilename string
```

Comment:
why store this as a separate file vs a key in the state file?

Reply:
it is generally written by the install script, so I put it in its own file to avoid JSON editing with unix text processing tools :)

Reply:
gotcha. that makes good sense :)